pkg/config/analysis/analyzers/deployment/services.go

```diff
@@ -93,6 +94,7 @@ func (s *ServiceAssociationAnalyzer) analyzeDeploymentPortProtocol(r *resource.I
 			for protocol := range protMap {
 				svcNames = append(svcNames, protMap[protocol]...)
 			}
+			sort.Strings(svcNames)
```

Comment:
It will be more convenient to sort it in the output layer instead of different analyzers. See https://github.com/istio/istio/blob/master/pkg/config/analysis/local/istiod_analyze.go#L176

Reply:
this's nothing about message order, it's about params order.